lms/static/scripts/frontend_apps/components/test/LMSGrader-test.js

```diff
@@ -84,61 +91,73 @@ describe('LMSGrader', () => {
     );
   });
 
-  it('passes a default value of "0" to onChangeSelectedUser when no a student is selected', () => {
+  it('does not set a focus user by default', async () => {
     renderGrader();
-    assert.isTrue(fakeOnChange.calledWithExactly('0'));
+    await fakeSidebarWindow;
+    assert.isTrue(
+      fakeRpcCall.calledOnceWithExactly(
+        'fake window',
+        'fake origin',
+        'changeFocusModeUser',
+        [
+          {
+            username: undefined,
+            displayName: undefined,
+          },
+        ]
+      )
+    );
   });
-
-  it('passes the unique userid to onChangeSelectedUser when a student is selected', () => {
+  it('sets the focused user if a valid index one is selected', async () => {
```

Comment:
"valid index one"? Did you mean "sets the focused user when a valid index is passed"?